code/DDSCodeTester.cpp

```diff
@@ -6225,6 +6251,39 @@ bool dds_rosbag_example()
     return true;
 }
 
+
+void pubsub_api_example_create_entities()
+{
+    //PUBSUB_API_CREATE_PARTICIPANT
+    DomainParticipant* participant =
+    DomainParticipantFactory::get_instance()->create_participant(0, PARTICIPANT_QOS_DEFAULT);
+    //!--
+
+    //PUBSUB_API_CREATE_PUBLISHER
+    Publisher* publisher = participant->create_publisher(PUBLISHER_QOS_DEFAULT);
+    //!--
+
+    //CREATE TOPIC
+    Topic* custom_topic =
+    participant->create_topic("HelloWorldTopic", "HelloWorld", TOPIC_QOS_DEFAULT);
+    //!--
+
+    //PUBSUB_API_CREATE_DATAWRITER
+    DataWriter* data_writer =
+        publisher->create_datawriter(custom_topic, DATAWRITER_QOS_DEFAULT);
+    //!--
```

Comment:
These code snippets seem to not be used anymore.
Shall we delete them?

Reply:
It's true that those snippets in particular are not used but `PUBSUB_API_WRITE_SAMPLE` is used, so I think it's better if we keep them because we still need that code to make it compile and also in case somebody else wants to use it.

Reply:
Makes sense, ty!